src/common/base/src/lib.rs

```diff
@@ -75,3 +75,5 @@ impl Plugins {
         binding.is_empty()
     }
 }
+
+pub type PluginsRef = Arc<Plugins>;
```

Comment:
can we just use `Plugins` instead `Arc<Plugins>` . because `Plugins` is a wrapper of Arc
I'm not sure.

Reply:
Actually we can, since we are not adding more fields to the struct. Let `Plugins` derive `Clone` and we can treat it like an `Arc`. 👍